### PR TITLE
FIX: Fallback to gzip compression if brotli isn't supported

### PR DESCRIPTION
**Pull request by OsamaSayegh**

### CODE REVIEW

File: spec/requests/static_controller_spec.rb [hunk 1]
```diff
@@ -120,6 +120,28 @@
     end
   end
 
+  context '#cdn_asset' do
```
Comment (ZogStriP) [inline spec/requests/static_controller_spec.rb]: How is this relevant to the changes made in `application_helper.rb`?

Looks like this is testing brotli assets when the other changes are about falling back to gzip?

Comment (OsamaSayegh) [reply spec/requests/static_controller_spec.rb]: Oh I just noticed we were missing a test for `#cdn_asset` so I added one. (also gzip and brotlit assets go through the same exact code path in this endpoint so I didn't duplicate this test for gzip)

Comment (ZogStriP) [reply spec/requests/static_controller_spec.rb]: I see, next time, it would be better to have that in a different commit ;)
